### DISCUSSION TITLE
keymap 豎屏適配有問題

### DISCUSSION THREAD

**Discussion by imac8t** [ayamir/nvimdots#1370] [General]
![image](https://github.com/user-attachments/assets/076012cb-4d8e-4b1c-9be6-e4dfbe2d228f)
看不全啊！

**Comment by ayamir** [comment]
You can adjust the width of it in `lua/keymap/helper.lua`:
https://github.com/ayamir/nvimdots/blob/08f2dda6833758237852931db7e1c7ca590cfadb/lua/keymap/helpers.lua#L7
